src/coreEnforcer.ts

```diff
@@ -395,8 +394,7 @@ export class CoreEnforcer {
 
       expression = this.getExpression(asyncCompile, expString);
       const context = { ...parameters, ...functions };
-      const result = asyncCompile ? await expression(context) : expression(context);
-
+      const result = asyncCompile ? yield expression(context) : expression(context);
```

Comment:
The result type is `{done: boolean, vaule: Promise<boolean> | boolean}`

Reply:
Thanks for your advice